aws/ecs/cloudwatch_events.tf

```diff
@@ -0,0 +1,96 @@
+#
+# Lambda: capture ECS events
+#
+resource "aws_lambda_function" "ecs_events" {
+  # checkov:skip=CKV_AWS_50: X-ray tracing not required for this function
+  # checkov:skip=CKV_AWS_115: No function-level concurrent execution limit required
+  # checkov:skip=CKV_AWS_116: No Dead Letter Queue required
+  # checkov:skip=CKV_AWS_117: VPC not required for this Lambda
+
+  function_name = "ecs_events"
+  description   = "Lambda function to write ECS events to a CloudWatch log group."
+
+  filename    = data.archive_file.ecs_events.output_path
+  handler     = "ecs_events.lambda_handler"
+  runtime     = "python3.9"
+  timeout     = 30
+  memory_size = 512
```

Comment:
we can probably drop this to the default of 128MB

Reply:
good idea - I've adjusted it